README.md

```diff
@@ -82,6 +84,52 @@ When this element is rendered, the DOM will look something like the following:
 
 This is the utility that web components provide when using Custom Elements and the Shadow DOM.
 
+Skate also allows **turning off Shadow DOM** if you don't wanna use it for various particular reasons. You can turn it off via `get renderRoot()` override:
+
+> NOTE: by turning off Shadow DOM you cannot use <slot/> content projection anymore by default, further tweaks needs to be applied
```

Comment:
It would be great to provide also solution how to achieve "slotting" without Shadow DOM. You mentioned that you have a solution for this, can you provide it please ?